database/mysql-schema.sql

```diff
@@ -0,0 +1,57 @@
+DROP DATABASE IF EXISTS events;
+CREATE DATABASE events;
+
+use events;
+
+CREATE TABLE org (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  org_name VARCHAR(255),
+  org_private BOOLEAN
+);
+
+CREATE TABLE series (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  series_description VARCHAR(255),
+  day_of_week VARCHAR(255),
+  series_interval INT
+);
+
+CREATE TABLE person (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  first_name VARCHAR(255),
+  last_name VARCHAR(255),
+  -- this was a holdover from the legacy database (MongoDB), where the identifier for each person was a string like `m0` `m145` `m487` etc...
+  identifier VARCHAR(255)
+);
+
+CREATE TABLE event (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  title VARCHAR(255),
+  local_date_time DATETIME,
+  org_id INT,
+  FOREIGN KEY (org_id)
+    REFERENCES org (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE,
+  series_id INT,
+  FOREIGN KEY (series_id)
+    REFERENCES series (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE
+);
+
+CREATE TABLE org_person (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  org_id INT,
+  FOREIGN KEY (org_id)
+    REFERENCES org (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE,
+  person_id INT,
+  FOREIGN KEY (person_id)
+    REFERENCES person (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE,
+  founder BOOLEAN,
```

Comment:
I thought sql didn't have a BOOLEAN data type?  I could be wrong but I though BIT was how you set true/false values.

Reply:
[BOOLEAN](https://www.mysqltutorial.org/mysql-boolean/) seems to be working for me... They are stored as 0s or 1s, which is neat.